datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -724,4 +724,42 @@ CREATE TABLE dcsa_im_v3_0.ebl_solution_provider_type (
 --- DDT-948
 ALTER TABLE dcsa_im_v3_0.equipment_event ADD utilized_transport_equipment_id uuid NULL REFERENCES dcsa_im_v3_0.utilized_transport_equipment(id);
 
+
+
+--- DDT-1017
+DROP TABLE IF EXISTS dcsa_im_v3_0.service_schedule CASCADE;
+CREATE TABLE dcsa_im_v3_0.service_schedule (
+    id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+    service_id uuid NOT NULL REFERENCES dcsa_im_v3_0.service (id),
+    carrier_id uuid NOT NULL REFERENCES dcsa_im_v3_0.carrier (id)
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.service_schedule_vessel_transportevent CASCADE;
+CREATE TABLE dcsa_im_v3_0.service_schedule_vessel_transportevent (
+    service_schedule_id uuid NOT NULL REFERENCES dcsa_im_v3_0.service_schedule (id),
+    vessel_id uuid NOT NULL REFERENCES dcsa_im_v3_0.vessel (id),

```

Comment:
I think the `service_schedule` needs to be a `vessel_schedule` and include the `vessel_id` (moving it up from `service_schedule_vessel_transportevent` which then could do with a rename to something like `vessel_schedule_terminal_visits` - take that naming suggestion with a grain of salt).

The logic we want to capture is:

- One service JOINS with
- 1-or-more vessel-schedules. Each of these then JOINS with
- 1-or-more terminal call visits

Reply:
The task call for a service_schedule table, not a vessel_schedule, likewise we are supposed to implement create GET service-schedules endpoint.

@HenrikHL do you have an opinion here? Should we rename service_schedule to vessel_schedule?